examples/calculate_rice_score.py

```diff
@@ -0,0 +1,126 @@
+# Reference: https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/
+from python_http_client import Client
+import os
+import json
+from datetime import datetime
+from email.utils import parsedate_tz, mktime_tz
+
+client = Client(host="http://{}".format(os.environ.get('DX_IP')))
```

Comment:
I see this host used a lot. Should it just be the default?

Reply:
Good idea.